client-course-schedulizer/public/dataInterfaces.tsx

```diff
@@ -0,0 +1,76 @@
+enum Semester {
+  Fall = "FA",
+  Interim = "IN", // TODO: Remove?
+  Spring = "SP",
+  Summer = "SU", // TODO: Is this a thing?
```

Comment:
I think it would be good to have these for now. However, by looking at the [new schedule](https://calvin.edu/dA/66abbcbe5f/Conceptual%20draft%20of%202021-2022%20academic%20calendar.pdf?language_id=1), does our current data model account for the four intensives, half-semester classes, and full-sesmeter classes? We might need to add some fields somewhere. Any suggestions?

Reply:
That's a good graphic, I didn't know the plan was so... complex. I wonder if we would just have to change the `Half` enum to have `Full`, `Session 1`, `Session 2`, `Intensive A`, `Intensive B`, `Intensive C`, and `Intensive D` options? Is this something we should change before the January scheduling?

Reply:
Hmm, I think we can leave it as-is for now, and circle back around if we need to update it.